### DISCUSSION TITLE
Agendas clarification

### DISCUSSION THREAD

**Discussion by phillipfung3** [bafolts/terraforming-mars#6769] [General]
so in agendas, there are a few policy involve global parameter, currently the pictogram is showing ocean, oxygen and temperature, but seems like it apply for Venus bar as well
so it get me to think does it work on moon bars as well? the wiki didn't clarify on these
and it might be helpful to either clarify on wiki or change the pictograms to reduce confusion

**Comment by kberg** [comment]
Can you please clarify which policies you mean?

**Reply by phillipfung3** [reply]
![image](https://github.com/terraforming-mars/terraforming-mars/assets/172607360/c12cc45b-31fe-496d-b0ce-aeaa48017bed)
scientist policy 3 (btw did it just got changed to draw and discard one in the wiki? I remember it was draw 1 just like the pictogram shows) 
![image](https://github.com/terraforming-mars/terraforming-mars/assets/172607360/1450d410-6a0c-450c-a5bb-3a7b11b1371b)

===============================================================================
![image](https://github.com/terraforming-mars/terraforming-mars/assets/172607360/e6b0fe2d-b4e9-4f9d-8ba4-2b95305b6274)
reds policy 3, checked that it currently work on everything including moon
![image](https://github.com/terraforming-mars/terraforming-mars/assets/172607360/85d2e9e8-7af2-43a8-945a-f40905360f2b)

================================================================================
![image](https://github.com/terraforming-mars/terraforming-mars/assets/172607360/1f5e26c8-5863-4ae5-b401-5c8b1cead716)
reds policy 4, yet it works on mars bars (including venus) but not moon, interesting
![image](https://github.com/terraforming-mars/terraforming-mars/assets/172607360/f126b8d5-ae15-44c3-9353-d2d2b1e20213)
![image](https://github.com/terraforming-mars/terraforming-mars/assets/172607360/533d6c04-ad82-4aef-840c-ca990d206e57)

**Comment by kberg** [comment]
It does apply to those. I'll add something to the wiki. Thanks.
